src/field/arena.ts

```diff
@@ -827,6 +827,36 @@ export class Arena {
     return !!tag;
   }
 
+  /**
+   * Find and remove all {@linkcode ArenaTag}s with the given tag types on the given side of the field.
+   * @param tagTypes - The {@linkcode ArenaTagType}s to remove
+   * @param side - The {@linkcode ArenaTagSide} to remove the tags from (for side-based tags), or {@linkcode ArenaTagSide.BOTH}
+   * to clear all tags on either side of the field
+   * @param quiet - Whether to suppress removal messages from currently-present tags; default `false`
+   * @todo Review the other tag manipulation functions to see if they can be migrated towards using this (more efficient)
+   */
+  public removeTagsOnSide(
+    tagTypes: ArenaTagType[] | ReadonlyArray<ArenaTagType>,
```

Comment:
```suggestion
    tagTypes: ArenaTagType[] | readonly ArenaTagType[],
```
I guess we don't have the lint enabled that enforces consistency for this.